hack/add-cluster.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
doesn't seem like this script adds a lot more functionalities than calling out register_argocd_cluster?
Is this needed as a fully separated script?

Reply:
It is just a convenience script. You'd have to source the functions script and call it instead. It is easier and less polluting of your local shell to just invoke a separate script if you want to add another cluster to your setup.

Reply:
Would it make more sense to invoke this script in the `setup.sh` then rather than using the `register_argocd_cluster`?